README.md

```diff
@@ -161,16 +161,27 @@ These are advanced techniques requiring a good understanding of tiptap internals
 
 ## Preserving additional tags and attributes
 
-The default `sanitize-html` configuration of Apostrophe does not save `colspan` and `rowspan` attributes, which the table editor of this module can produce. You will want to use a configuration like this. Note that since we are touching `allowedAttributes`, we must list *all* of the attributes that we believe should be allowed on any tag, not just the ones for table cells. If you fail to address this, you will lose the `href` attributes of your links.
+There is a default `sanitize-html` configuration in this package to save `colspan` and `rowspan` attributes, which the table editor of this module can produce.
 
 ```javascript
+sanitizeHtml: {
+  allowedAttributes: {
+    a: ['href', 'name', 'target'],
+    td: ['colspan', 'rowspan'],
+    th: ['colspan', 'rowspan']
+  }
+}
+```
+
+You can add your own configuration. It will be merged with the default one. For instance:
```

Comment:
sanitize-html configuration is not merged, if `allowedTags` is specified you must list every allowed tag, if `allowedAttributes` is specified you must specify every allowed attribute, etc.

Reply:
Ok